src/pages/Share/SubmitDetailsPage.tsx

```diff
@@ -173,6 +173,7 @@ function SubmitDetailsPage({
                         receiptFilename={getFileName(currentAttachment?.content ?? '')}
                         reportID={reportOrAccountID}
                         shouldShowSmartScanFields={false}
+                        isDistanceRequest={false}
```

Comment:
Note for future: we can't create distance in share flow, so this is always false